esphome/components/esp32_ble_client/ble_client_base.cpp

```diff
@@ -30,23 +30,25 @@ void BLEClientBase::loop() {
 
 float BLEClientBase::get_setup_priority() const { return setup_priority::AFTER_BLUETOOTH; }
 
+void BLEClientBase::found_device(uint64_t addr, esp_ble_addr_type_t address_type) {
```

Comment:
```suggestion
void BLEClientBase::set_found_device(uint64_t addr, esp_ble_addr_type_t address_type) {
```

I think `set_` is better for something that mutates state depending on input